js_assets/Question.vue

```diff
@@ -107,6 +107,12 @@
                 })
                 this.answerVal = "";
             },
+            logAttempt(answer) {
```

Comment:
If we're going to put logAttempt as a route separate from the answer, it would be good to only log the attempt if the answer route returned successfully.  So, either we move this logic into the .then() method from the answer promise or we can create a function for logging Attempts and put that in the answer view.

Reply:
this is genius! I was trying to include it in the answer method, I just had my axios request in the wrong place